### ISSUE TITLE
Readme.md points to outdated prereqs

### ISSUE THREAD

**Issue by nlburgin** [TASVideos/BizHawk#1886]
EDIT: Most of this post can probably be skipped, it was made without an awareness that the dependencies had been updated.

I'm back to TASing after a hiatus, and I wanted to test the patch that was supposed to possibly fix #1646. So I downloaded the latest dev build 7fb89aed18b5da7ad6befe9156a7a8e94c454dbe but it crashed on startup. The circumstances of the crash don't seem to match any of the other crash-at-startup issues I've seen posted here:

![Screenshot_2020-03-20_12-22-50](https://user-images.githubusercontent.com/30382228/77184110-b3635100-6aa5-11ea-9e7f-c3f5ef729d16.png)

Here are the "problem details":

```
Problem signature:
  Problem Event Name:	CLR20r3
  Problem Signature 01:	EmuHawk.exe
  Problem Signature 02:	2.4.0.16086
  Problem Signature 03:	5e742c6a
  Problem Signature 04:	EmuHawk
  Problem Signature 05:	2.4.0.16086
  Problem Signature 06:	5e742c6a
  Problem Signature 07:	b48
  Problem Signature 08:	0
  Problem Signature 09:	System.IO.FileNotFoundException
  OS Version:	6.2.9200.2.0.0.768.101
  Locale ID:	1033
  Additional Information 1:	f536
  Additional Information 2:	f536a6bcb56ab99a9a84961f6978269a
  Additional Information 3:	88f1
  Additional Information 4:	88f1a8a8df2ce8b7a9a2dbd9635122f2
```

I figured this was probably something transient with the latest build, so I went back a month to the version where the patch I actually wanted to test was introduced: ca33a0ac9ddd0275569f343152a5b048283a8700

But, it gave the same exact error.

So I went back another month to Release 2.4, fearing the same... but it ran just fine, actually.

I really don't know what's causing this error, or how others might reproduce it. If it were happening to everyone in every build for 2 months, it seems too obvious to miss; I'm assuming these versions are at least running for the devs before being committed. However, it also doesn't *seem* like it could be a problem with my own system, since I can run other builds and AFAIK the dependencies haven't changed.

And it doesn't seem like it could be an inherent difference between dev builds and releases, since a much older dev build I had lying around, 0753c9d92 , still works just fine.

I can't figure out what's going on. I did notice the crash dialog mentioned a "File Not Found Exception"; maybe there was a regression in Appveyor's configuration that causes it to leave out an important file when it packages dev builds? I'm grasping at straws here but that's all I can think of.

**Comment by nlburgin** [comment]
The only other possibility I can think of is maybe an undocumented (or even unintentional) dependency change that was introduced sometime after 2.4? Which went unnoticed by the devs due to them having up to date systems? IDRK.

**Comment by alyosha-tas** [comment]
Have you run the most recent pre-req installer? I t was updated recently.

**Comment by zeromus** [comment]
Whenever something crashes due to "file not found", run process monitor and look for missing files and include those with the bug report

**Comment by nlburgin** [comment]
>Have you run the most recent pre-req installer? I t was updated recently.

Okay, that's almost certainly the issue, then. I wasn't aware of that because the link in Readme.md points [here](https://github.com/TASVideos/BizHawk-Prereqs/releases/tag/2.1) instead of [here](https://github.com/TASVideos/BizHawk-Prereqs/releases).

**Comment by zeromus** [comment]
Like i SAID, yoshi
https://github.com/TASVideos/BizHawk/commit/cab5df6c8720881e0fbdf49a4db50584c4f63b26#diff-04c6e90faac2675aa89e2176d2eec7d8
People will be confused no matter what, so you may as well choose to make the link which doesn't depend on someone to update it later

**Comment by YoshiRulz** [comment]
I can't imagine the new prereq installer will fix this where the old one didn't, simply because *it does less*. The only *addition* was .NET 4.8 and if that was the cause, OP needs to update Windows.

**Comment by zeromus** [comment]
I believe the newer-than-UCRT "Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019" is required for correct functioning on windows 10, as of MS-forced windows update version ?????????????? 
so newer prereqs which "do" less are actually doing less in a way that works on more systems

**Comment by nlburgin** [comment]
>OP needs to update Windows.

Oh, I sure hope not. 

I'm running BK on a janky old version of Windows 8 running inside a Virtualbox VM that I use for no other purpose and hasn't been connected to the internet at all in a long time. Attempting to update it sounds like a recipe for pain.

**Comment by YoshiRulz** [comment]
Fixed in 48658e2c3.
